app/graphql/types/organizations/update_organization_input.rb

```diff
@@ -29,8 +29,8 @@ class UpdateOrganizationInput < BaseInputObject
       argument :document_number_prefix, String, required: false
       argument :document_numbering, Types::Organizations::DocumentNumberingEnum, required: false
 
-      argument :billing_configuration, Types::Organizations::BillingConfigurationInput, required: false
-      argument :email_settings, [Types::Organizations::EmailSettingsEnum], required: false
+      argument :billing_configuration, Types::Organizations::BillingConfigurationInput, required: false, permission: 'organization:invoices:view' # rubocop:disable Layout/LineLength
```

Comment:
Same question regarding line length

Reply:
Here it's a longer, it could be multiline but I prefer to keep the `argument` flow 🙏

Reply:
Well, I think we should avoid using `rubocop:disable` as much as possible and I don't think it's an issue to have multiple lines here.

That said, not a blocker for me :)

Reply:
Yes, I agree it should be some sort of last resort. If we didn't decide to move to standardrb, I think I'd have broken the line down.